mkdocs.yml

```diff
@@ -1,5 +1,5 @@
 site_name: avalanche-cli
-site_description: Avalanche-CLI is a command-line interface for building, deploying, and maintaining Avalanche Subnets. It can be used to support the entire Subnet development lifecycle from initial prototyping to production deployments. Avalanche-CLI is available for Linux and Mac, and is open source on GitHub.
+site_description: Avalanche-CLI is a command-line interface for building, deploying, and maintaining Avalanche Blockchains. It can be used to support the entire Blockchain development lifecycle from initial prototyping to production deployments. Avalanche-CLI is available for Linux and Mac, and is open source on GitHub.
 nav:
   - Introduction: index.md
   - Ledger Simulator: ledger-simulator.md
```

Comment:
What is this .yml serving? if it is for a deprecated github page, we may as well delete this file since the CLI docs are hosted in the Dev docs